src/react/databrowser/objects/__tests__/ObjectUpload.test.jsx

```diff
@@ -0,0 +1,133 @@
+import * as s3object from '../../../actions/s3object';
+import ObjectUpload, { FileList, NoFile } from '../ObjectUpload';
+import { BUCKET_NAME } from '../../../actions/__tests__/utils/testUtil';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { reduxMount } from '../../../utils/test';
+
+describe('ObjectUpload', () => {
+    const closeObjectUploadModalSpy = jest.spyOn(s3object, 'closeObjectUploadModal');
```

Comment:
Is there a difference between a "mocked method" with 'Spy' or 'Mock suffix?

Reply:
No there are no differences. Do you want me to change "Spy" suffix by "Mock" to be consistent with others tests ?

Reply:
Yes please :)

Reply:
done